cookbooks/fb_systemd/recipes/networkd.rb

```diff
@@ -23,6 +23,7 @@
 }.each do |svc|
   service svc do
     only_if { node['fb_systemd']['networkd']['enable'] }
+    subscribes :restart, 'package[systemd packages]', :immediately
```

Comment:
Won't this cause issues if `manage_systemd_packages` is false? It might be better to do this as a notify instead

Reply:
Why would it? that resource still exists (its not in an `if`, but never actually fires `it has an `only_if`. This should work fine. However, I certainly can hang all of these off the package, I was following suite since you have no notifies on that package... and since you subscribe to it from `boot.rb`, I was following suite. Since someone might carve these things up and not include specific recipes and thus not *have* these various services, the `notifies` is a bit more brittle, I think. `subscribes` allows the thing it's subscribing to to not exist.

Reply:
Yeah, I think you're right actually.